ldclient.go

```diff
@@ -342,7 +342,7 @@ func MakeCustomClient(sdkKey string, config Config, waitFor time.Duration) (*LDC
 		for {
 			select {
 			case <-closeWhenReady:
-				if client.dataSystem.DataAvailability() != datasystem.Refreshed {
+				if client.dataSystem.DataAvailability() != datasystem.Defaults {
```

Comment:
(self-reminder): I can't make this change as it would be a modification of existing behavior.